libs/core/langchain_core/tracers/langchain.py

```diff
@@ -123,8 +123,8 @@ def _start_trace(self, run: Run) -> None:
                 run.tags = self.tags.copy()
 
         super()._start_trace(run)
-        if run._client is None:
-            run._client = self.client  # type: ignore[misc]
+        if run.ls_client is None:
```

Comment:
I believe that this breaks with current constraints on the langsmith client. ls_client was available as of 0.1.126. Bumping the requirement

Reply:
Pushed a PR to update the min constraint and updated the uv lock file